### ISSUE TITLE
[Bug]: 2.7.3 broke coreutils on x86 MacOS

### ISSUE THREAD

**Issue by timbess** [aspect-build/bazel-lib#838]
### What happened?

https://github.com/aspect-build/bazel-lib/blob/1697a3275becc29d0c002f6ef18bd63885b4d758/lib/private/coreutils_toolchain.bzl#L54

This ends up generating this URL `https://github.com/uutils/coreutils/releases/download/0.0.26/coreutils-0.0.23-x86_64-apple-darwin.tar.gz` which is not valid.

### Version

Development (host) and target OS/architectures:

Output of `bazel --version`:
```
❯ bazel --version
bazel 7.1.2
```

Version of the Aspect rules, or other relevant rules from your
`WORKSPACE` or `MODULE.bazel` file:
```
bazel_dep(name = "aspect_bazel_lib", version = "2.7.3")
```

Language(s) and/or frameworks involved: rust, python, js, c++, c

### Any other information?

Seems like the newest release of coreutils dropped x86 MacOS? Not sure if that was on purpose, but I assume that's why this version was left at `0.0.23`, but `0.0.26` is prepended before the name of the file which breaks the URL.

**Comment by plobsing** [comment]
Was this fixed by https://github.com/aspect-build/bazel-lib/pull/853 ?

**Comment by timbess** [comment]
> Was this fixed by #853 ?

Yeah pretty sure that should fix it. Can't confirm right now, but I can reopen it we bump into it again 👍
